game/graphics/pipelines/opengl.cpp

```diff
@@ -389,12 +402,24 @@ void GLDisplay::on_iconify(GLFWwindow* /*window*/, int iconified) {
 }
 
 namespace {
-std::string make_output_file_name(const std::string& file_name) {
-  file_util::create_dir_if_needed(file_util::get_file_path({"gfx_dumps"}));
-  return file_util::get_file_path({"gfx_dumps", file_name});
+std::string make_full_screenshot_output_file_path(const std::string& file_name) {
+  file_util::create_dir_if_needed(file_util::get_file_path({"screenshots"}));
+  return file_util::get_file_path({"screenshots", file_name});
 }
 }  // namespace
 
+static std::string get_current_timestamp() {
+  auto current_time = std::time(nullptr);
+  auto local_current_time = *std::localtime(&current_time);
+  std::ostringstream oss;
+  oss << std::put_time(&local_current_time, "%Y_%m_%d_%H_%M_%S");
+  return oss.str();
+}
```

Comment:
Looking at `sceCdReadClock`, we do the following to get the time:

```cpp
  time_t t = time(0);
  std::tm* date = localtime(&t);
```

Stringstreams are kinda janky and we haven't used them anywhere, so it might be better to just format the string using that code.

Reply:
I opted to use strftime instead.  fmt has date formatting support but we are missing "fmt/chrono.h" https://fmt.dev/latest/api.html